tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,
+envlist =
           py36,
           py37,
           py38,
```

Comment:
Huh, we did not even have retrictions on pytest here for 2.7...

Reply:
Pip knows how to handle `python_requires`.